packages/transactional-adapters/transactional-adapter-pg-promise/package.json

```diff
@@ -62,8 +62,5 @@
         "ts-node": "^10.8.1",
         "tsconfig-paths": "^4.0.0",
         "typescript": "~4.8.0"
-    },
-    "dependencies": {
-        "install-peers": "^1.0.4"
```

Comment:
Please run `yarn` and commit the new `yarn.lock`. The CI is failing because of it.